### PR TITLE
Add a configurable standby mode that turns off the heater after a certain amount of time

### PR DESCRIPTION
**Pull request by fiendie**
* Add a new power section in the web interface
* Switch standby mode on or off, set the duration until the heater is turned off 
* Ways to exit the standby mode:
   * Brew detection
   * Toggle pid controller/steam mode from web interface
   * Set pidON/steamON to 1 via MQTT
* Timer counts down from the configured duration and is reset by state transitions to kBrew, kSteam and kBackflush

### CODE REVIEW

**Comment by LoQue90** [comment]
runs flawless for me. 
Shelly reports 1,1w in standby.
MQTT Standby topics opens new ways of automation!
thanks @fiendie 
Go for merge

File: src/Storage.h [hunk 1]
```diff
@@ -40,8 +40,10 @@ typedef enum
   STO_ITEM_WIFI_SSID,               // Wifi SSID
   STO_ITEM_WIFI_PASSWORD,           // Wifi password
   STO_ITEM_WEIGHTSETPOINT,          // Brew weight setpoint
-  STO_ITEM_RESERVED_28,             // reserved
-  STO_ITEM_RESERVED_29,             // reserved
+  STO_ITEM_STANDBY_MODE_ON,         // Enable tandby mode
+  STO_ITEM_STANDBY_MODE_TIME,       // Time until heater is turned off
+  STO_ITEM_RESERVED_30,             // reserved
+  STO_ITEM_RESERVED_21,             // reserved
```
[the remark this answers is not in the document]
Comment (kjyv) [reply src/Storage.h:46]: Also, do we really need to reserve memory slots here? We're usually erasing the flash anyway

[the remark this answers is not in the document]
Comment (LoQue90) [reply src/Storage.h:46]: as far as I remeber reserve memory slots is to keep be compatible with old 2.x software versions.
i think with 4.0 release we could go for a breaking change and make it nessecary to erase flash and reconfigure.
so maybe we also rearange the memory and sort it in like system settings, connection settings, pid params usw and get rid of the reservations.

[the remark this answers is not in the document]
Comment (kjyv) [reply src/Storage.h:46]: I don't think there is much compatibility left :)
IMO, the idea is that you can update the software with new parameters without having to erase flash this way. But only if the values in other fields have the sames meaning, types etc. and only new fields are added into reserved slots

**Pull request merged**